src/openklant/components/klantinteracties/api/filterset/digitaal_adres.py

```diff
@@ -0,0 +1,87 @@
+import uuid
+
+from django.utils.translation import gettext_lazy as _
+
+from django_filters.rest_framework import FilterSet, filters
+
+from openklant.components.klantinteracties.models.digitaal_adres import DigitaalAdres
+
+
+class DigitaalAdresFilterSet(FilterSet):
+    verstrekt_door_betrokkene__uuid = filters.UUIDFilter(
+        help_text=_(
+            "Zoek digitaal adres(sen) object(s) op basis van de betrokkene uuid."
```

Comment:
Applies to the filters below as well
```suggestion
            "Zoek digitaal adres(sen) object(en) op basis van de betrokkene uuid."
```